mrsl_quadrotor_description/urdf/sensors/range_beam.urdf.xacro

```diff
@@ -19,7 +19,7 @@
         <visual>
           <origin xyz="0 0 0" rpy="0 ${M_PI/2} 0" />
           <geometry>
-            <cylinder radius="0.01" length="0.1"/>
+            <cylinder radius="0.01" length="0.01"/>
```

Comment:
remove extra `_frame` from this file line 10, 13, 32

Reply:
I removed the extra _frame from lines 10, 13, 32
I have tested the code after removing and it works fine
I have pushed after the testing